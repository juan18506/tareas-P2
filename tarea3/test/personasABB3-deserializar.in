crearPila

# Construyo Pila
crearPersona 20 20 Caroline
apilar
liberarPersona

crearPersona 2 2 Romina
apilar
liberarPersona

crearPersona 7 7 Carlos
apilar
liberarPersona

# Transformo lista en arbol
deserializarArbolPersonas

Fin