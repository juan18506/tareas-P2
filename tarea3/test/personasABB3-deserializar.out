1>2>#  Construyo Pila.
3>4>5>6>7>8>9>10>11>12>#  Transformo lista en arbol.
13>Persona 2:  Romina, 2 años
Persona 7:  Carlos, 7 años
Persona 20:  Caroline, 20 años
14>Fin.
